src/code/game.c

```diff
@@ -364,28 +365,28 @@ void GameState_Realloc(GameState* gameState, size_t size) {
 
     THA_Destroy(&gameState->tha);
     GameAlloc_Free(alloc, thaStart);
-    PRINTF("ハイラル一時解放!!\n"); // "Hyrule temporarily released!!"
+    PRINTF(T("ハイラル一時解放!!\n", "Hyrule temporarily released!!\n"));
     SystemArena_GetSizes(&systemMaxFree, &systemFree, &systemAlloc);
     if ((systemMaxFree - 0x10) < size) {
         PRINTF("%c", BEL);
         PRINTF(VT_FGCOL(RED));
 
-        // "Not enough memory. Change the hyral size to the largest possible value"
-        PRINTF("メモリが足りません。ハイラルサイズを可能な最大値に変更します\n");
+        PRINTF(T("メモリが足りません。ハイラルサイズを可能な最大値に変更します\n",
+                 "Not enough memory. Change Hyrule size to maximum possible value\n"));
         PRINTF("(hyral=%08x max=%08x free=%08x alloc=%08x)\n", size, systemMaxFree, systemFree, systemAlloc);
         PRINTF(VT_RST);
         size = systemMaxFree - 0x10;
     }
 
-    PRINTF("ハイラル再確保 サイズ＝%u バイト\n", size); // "Hyral reallocate size = %u bytes"
+    PRINTF(T("ハイラル再確保 サイズ＝%u バイト\n", "Hyral reallocate size = %u bytes\n"), size);
```

Comment:
we can change these google translate "Hyral"s to Hyrule, some of them are already correct even